osuosl/climesync
# Added option to disable config autoupdate

fixes issue #59 

## Changes in this PR.


- [X] Added "autoupdate_config" option to .climesyncrc
- [x] Documented the new configuration option

## Testing this PR.


1. Run `nosetests`
2. If ~/.climesyncrc doesn't exist, run the program and log in to generate one
3. Edit ~/.climesyncrc in your favorite text editor and add the line `autoupdate_config = false`
4. Run climesync again and log in as a different user
5. See that you aren't asked if you want to add the new username and password to the config
6. Edit the config again and set `autoupdate_config = true`
7. Run climesync once again and log in as a different user
8. See that you are asked if you want to update the config again

@osuosl/devs
